### ISSUE TITLE
Add Shared Coordinates config to action config (or similarly named)

### ISSUE THREAD

**Issue by rwemay** [BHoM/Revit_Toolkit#687]
#### Description:

Important to be able to pull objects in shared coordinates system rather than origin coordinates, as a lot of models/disciplines do not align their origin coordinates such that when elements are pulled from different models, they are in different spaces. This makes any coordination outside of Revit very difficult.

**Comment by pawelbaran** [comment]
Good point @rwemay - I guess this would become a part of Revit_Settings (settings on adapter level, not action level)?

**Comment by rwemay** [comment]
Not sure, I think it's an action config on pull since you might want to pull geometry in different coord systems with different pulls.

**Comment by pawelbaran** [comment]
Hmm wouldn't making multiple pulls with this setting quickly become cumbersome? Just asking, I will lean on your project experience with that one.

**Comment by FraserGreenroyd** [comment]
I guess you would leave it blank (or default to global coords) and only use it as a setting on selected pulls? It's also likely not a setting for the push (could be wrong), so putting it on action config makes most sense to me (for what it's worth)

**Comment by pawelbaran** [comment]
Hmm you are touching an important subject actually @FraserGreenroyd - I think this will be relevant for both pull and push. This would drag me even closer to suggesting linking it with general Revit settings instead of action configs (imagine having multiple pull/push/update actions in a single script, needing to remember to fix config for each of them).

My gut feeling is that the coordinate systems are set per project, not per action. But again, I believe @rwemay has much more experience with project work, so would happy to change my mind if explained well.

**Comment by rwemay** [comment]
I think would actually be useful to pull geometry in shared and origin coordinates for individual objects in side by side so you can check and visualise differences. Naturally you could create two adapters for that. We will want to be able to push based on SharedCoordinates too, but not sure it's that straight forward - I feel like we'd need more than just the geometry and that it's working in SharedCoordinates. There's also the 3rd set of coordinates.... derived from the project base point. 

I don't particularly mind it being on the adapter, other than us having previously loaded way too much in there! To me it's more intuitive as an adapter action - synonymous with import/export options in the Revit UI itself where you select the different coordinate systems. Don't forget I think we also want to pull the coordinate system information itself (apologies have not checked if that's implemented).

**Comment by pawelbaran** [comment]
We could probably hack the three coordinate systems out of Revit using `FilterByDBTypeName` or id-based requests. However, this could be exposed in a neater way for sure. I will give it a shot to map out the following:
1. Extraction of different coordinate systems and location
2. Smart way of setting these on Push/Pull

Cannot promise this will happen any time soon though (as long as not urgently needed) - apologies for that.

**Comment by rwemay** [comment]
No absolutely - I don't think this is as yet critical. Good to just get into the list/boards.
